lib/Twig/Node/Expression/Name.php

```diff
@@ -61,21 +61,19 @@ public function compile(Twig_Compiler $compiler)
                     ;
                 }
             } else {
-                // When Twig will require PHP 7.0, the Template::notFound() method
-                // will be removed and the code inlined like this:
-                // (function () { throw new Exception(...); })();
                 $compiler
                     ->raw('(isset($context[')
                     ->string($name)
                     ->raw(']) || array_key_exists(')
                     ->string($name)
                     ->raw(', $context) ? $context[')
                     ->string($name)
-                    ->raw('] : $this->notFound(')
+                    ->raw('] : (function () { throw new Twig_Error_Runtime(\'Variable ')
                     ->string($name)
```

Comment:
this is fragile, as ``->string($name)`` will output quotes, which may close the string started as ``'Variable `` if you switch the code to use single quotes rather than double quotes in the future.